wallet/wallet.go

```diff
@@ -42,6 +59,99 @@ type WalletEncryptionArgs struct {
 	Privileged       bool
 	PrivilegedReason string
 	SeekPermission   bool
+	Plaintext        []byte
+	Ciphertext       []byte
+}
```

Comment:
`WalletEncryptionArgs` is a base struct for all args which pass the derivation information. 
```
type WalletEncryptionArgs struct {
	ProtocolID       WalletProtocol
	KeyID            string
	Counterparty     WalletCounterparty
	Privileged       bool
	PrivilegedReason string
	SeekPermission   bool
}

type WalletEncryptArgs struct {
    	WalletEncryptionArgs
    	Plaintext        []byte
}

type WalletDecryptArgs struct {
    	WalletEncryptionArgs
    	Ciphertext        []byte
}
```